src/sentry/options/defaults.py

```diff
@@ -1793,6 +1793,10 @@
     "hybrid_cloud.disable_relative_upload_urls", default=False, flags=FLAG_AUTOMATOR_MODIFIABLE
 )
 
+# Flagpole Rollout
+register("features", default={}, flags=FLAG_AUTOMATOR_MODIFIABLE)
```

Comment:
Would we have all feature flags stored as keys in this dictionary? Another approach could be to have one option per feature flag, and prefix options with `features.` That would let us load feature data as required and not evict the entire feature cache when any feature changes.

Reply:
Hmm wouldn't this also require individually registering each feature option with a default as well? Either that or we fully relax the concept of pushing unknown options to sentry (or regex match against feature_*?)

Reply:
Revisiting this finally, I think a single features dictionary is fine to start. We'll work on splitting this into individual features as needed if we encounter perf issues retrieving or setting this monolithic features option.